SECURITY.md

```diff
@@ -1,11 +1,142 @@
+    1. In response to an email to [security@aptosfoundation.org](mailto:security@aptosfoundation.org), a member of the Aptos Labs will create a new draft security advisory for the incident at [https://github.com/aptos-labs/aptos-core/security/policy](https://github.com/aptos-labs/aptos-core/security/policy).
+    2. Add the reporter's GitHub account and relevant individuals to the draft security advisory.
+    3. Respond to the reporter by email, sharing a link to the draft security advisory.
+2. Responder should add appropriate content to the draft security advisory. To be eligible for a bug bounty, this includes:
+    1. A clear description of the issue and the impacted areas.
+    2. The code and the methodology to reproduce the underlying issue.
+    3. Discussion of potential remediations.
+3. Triage
+    1. Validate the issue.
+    2. Determine the criticality of the issue.
+    3. If this is not an issue but a bug, recommend to the submitter create an issue
+4. Resolve by building new docker containers that contain the fix and pushing to affected entities.
+
+## Bug Bounties
+
+Aptos Foundation offers bounties for security reports. Reports will be validated against the GitHub repository branch labeled "main" and no others. Moreover, any code within `main` but not in an actively deployed network or in a position to be deployed in such an environment, such as experimental or actively developed code, are out of scope for the bug bounty program. Production environments include the Aptos testnet and mainnet.
+
+The scope includes code within
+
+- [Aptos Core — main branch](https://github.com/aptos-labs/aptos-core/tree/main)
+- [Move — Aptos branch](https://github.com/move-language/move/tree/aptos)
+
+Aptos Foundation considers the following levels of severity:
+
+****************Critical — Up to $1,000,000 USD in APT tokens (locked for 12 months)****************
+
+- Direct loss of funds to users or protocols with minimal preconditions
+- Execute arbitrary system commands (RCE) on validator nodes
+- Vulnerabilities in the Proof of Stake system which directly compromise consensus
+- Unintended permanent chain split requiring hard fork (network partition requiring hardfork)
+- Permanent freezing, burning, or modification of funds (fix requires hardfork)
+
+****************High — Up to $100,000 USD in APT tokens (locked for 12 months)****************
+
+- Loss of funds with some pre-conditions
+- Interrupting blockchain progress or halting the network
+
+****************Medium — Up to $25,000 USD in APT tokens (locked for 12 months)****************
+
+- Denial of service issues which compromise the integrity or availability of the chain
+- Loss of funds with many preconditions
+- Ability to crash a production node with some pre-conditions
+
+## **Payment of Bug Bounties**
+
+- Bounties are currently awarded on a rolling/weekly basis and paid out within 30 days upon receipt successful KYC and payment contract.
```

Comment:
`receipt successful KYC and payment contract.` -> `receipt of successful KYC and payment contract.`